src/types/fixedtimezone.jl

```diff
@@ -95,3 +95,20 @@ end
 
 name(tz::FixedTimeZone) = tz.name
 rename(tz::FixedTimeZone, name::AbstractString) = FixedTimeZone(name, tz.offset)
+
+function Base.:(==)(a::FixedTimeZone, b::FixedTimeZone)
+    a.name == b.name && a.offset == b.offset
+end
+
+function Base.isequal(a::FixedTimeZone, b::FixedTimeZone)
+    return (
+        isequal(a.name, b.name) &&
+        isequal(a.offset, b.offset)
+    )
+end
+
+function Base.hash(tz::FixedTimeZone, h::UInt)
+    h = hash(tz.name, h)
+    h = hash(tz.offset, h)
+    return h
+end
```

Comment:
Including the `Symbol` in the hash ensures the resulting hash differs from other structs which happen to have the same fields.
```suggestion
    h = hash(:timezone, h)
    h = hash(tz.name, h)
    h = hash(tz.offset, h)
    return h
end

```